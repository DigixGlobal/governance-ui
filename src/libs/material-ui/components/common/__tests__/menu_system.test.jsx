import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import MenuSystem from '../menu_system';

const tabs = [{
  icon: 'tabIcon1',
  name: 'Tab 1',
  path: '/tab-1',
  exact: true,
}, {
  icon: 'tabIcon2',
  name: 'Tab 2',
  path: '/tab-2',
  exact: true,
}];

describe('<MenuSystem />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <MenuSystem
        tabs={tabs}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
