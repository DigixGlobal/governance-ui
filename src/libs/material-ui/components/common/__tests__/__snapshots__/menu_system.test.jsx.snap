// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuSystem /> renders correctly 1`] = `
<div>
  <Menu>
    <Container>
      <MenuItem
        active={true}
        content="Tab 1"
        exact={true}
        icon="tabIcon1"
        key="/tab-1"
        onClick={[Function]}
      />
      <MenuItem
        active={false}
        content="Tab 2"
        exact={true}
        icon="tabIcon2"
        key="/tab-2"
        onClick={[Function]}
      />
    </Container>
  </Menu>
  <Container
    style={
      Object {
        "marginTop": "1.5em",
      }
    }
  />
</div>
`;
